webviews/common/context.tsx

```diff
@@ -5,7 +5,7 @@
 
 import { createContext } from 'react';
 import { IComment } from '../../src/common/comment';
-import { EventType, isReviewEvent, ReviewEvent } from '../../src/common/timelineEvent';
+import { EventType, isReviewEvent, ReviewEvent, ReviewResolveInfo, TimelineEvent } from '../../src/common/timelineEvent';
```

Comment:
lint error with unused `ReviewResolveInfo`. I'll try to see why the build isn't catching this.